features/admin/dashboard_per_channel.feature

```diff
@@ -13,38 +13,50 @@ Feature: Statistics dashboard per channel
         And the store allows paying Offline
         And I am logged in as an administrator
 
-    @ui
+    @ui @no-api
     Scenario: Seeing basic statistics for the first channel by default
         Given 3 customers have fulfilled 4 orders placed for total of "$8,566.00" mostly "Onion" product
         And 2 more customers have fulfilled 2 orders placed for total of "$459.00" mostly "Banana" product
-        When I open administration dashboard
+        When I browse administration dashboard statistics
         Then I should see 4 new orders
         And I should see 5 new customers
         And there should be total sales of "$8,566.00"
         And the average order value should be "$2,141.50"
 
-    @ui
+    @api @ui
     Scenario: Changing channel in administration dashboard
         Given 4 customers have fulfilled 4 orders placed for total of "$5,241.00" mostly "Onion" product
         And 2 more customers have fulfilled 2 orders placed for total of "$459.00" mostly "Banana" product
         And 2 more customers have placed 3 orders for total of "$1,259.00" mostly "Banana" product
-        When I open administration dashboard
+        When I browse administration dashboard statistics for "Poland" channel
         And I choose "United States" channel
         Then I should see 2 new orders
         And I should see 8 new customers
         And there should be total sales of "$459.00"
         And the average order value should be "$229.50"
 
-    @ui
+    @api @ui
+    Scenario: Changing channel in administration dashboard
```

Comment:
Even though these scenarios may seem similar, I found them valuable because one consists only of fulfilled orders and the other is a mix of fulfilled and placed orders, which affects the final summary. I modified the scenario names to accurately reflect that fact